### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by coryschwartz**

### CODE REVIEW

**Comment by Robmat05** [comment]
Typo in 2nd paragraph,  first sentence. Should be Scales, not Scaleu

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# IPFS Testground infrastructure
+
+This repo contains scripts for setting up a kubernetes cluster for [ipfs testground](https://testground.ipfs.team)
+
+Using the kubernetes cluster runner on IPFS testground enables you to test p2p workloads at high scaleu. Testing at high volume is a pivotal component of developing rock solid distributed software. Using the kuberentes runner, testground can test workloads with 10k+ instances. While this is magnitudes smaller than the interplanetary-scale IPFS network, testing at this level has enabled us to find issues that would have been undetectable on smaller tests.
```
Comment (nonsense) [inline README.md:5]: `scale` typo.

Comment (coryschwartz) [reply README.md:5]: Doh! thanks.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The [IPFS](https://ipfs.io/) and [libp2p](https://libp2p.io/) projects have used these scripts and playbooks to deploy large-scale test infrastructure. By crafting test scenarios that exercise components at such scale, we have been able to run simulations, carry out attacks, perform benchmarks, and execute all kinds of test to validate correctness at scale.
+
+## Quick start
+
+<@coryschwartz>
+
+## Documentation
+
+<@coryschwartz: point to the relevant sections in GitBook>
```
Comment (raulk) [inline README.md:17]: @coryschwartz

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Using the `cluster:k8s` runner of Testground enables you to test distributed/p2p systems at large scales. Testing at large scale is an essential component of developing rock-solid distributed software.
+
+The `cluster:k8s` Testground runner is capable of launching test workloads comprising 10k+ instances, and we aim to reach 100k at some point.
+
+The [IPFS](https://ipfs.io/) and [libp2p](https://libp2p.io/) projects have used these scripts and playbooks to deploy large-scale test infrastructure. By crafting test scenarios that exercise components at such scale, we have been able to run simulations, carry out attacks, perform benchmarks, and execute all kinds of test to validate correctness at scale.
+
+## Quick start
+
+<@coryschwartz>
```
Comment (raulk) [inline README.md:13]: @coryschwartz

**Pull request merged**